react-native/ios/Keybase/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder {
 
       #if DEBUG
         if let reactHost = AppDefault.ReactHost.stringValue {
-          return NSURL(string: "http://\(reactHost)/shared/index.bundle?platform=ios&dev=true")
```

Comment:
Yes. This change is what I meant by the parity above. We were running a dev iOS environment that wasn't necessarily exactly the same as the prod one (if `index.ios.js` had anything in it then it would appear in prod but not dev). With this change the prod and dev envs will be the same.